### ISSUE TITLE
Hirschfeld chapter : Amanda

### ISSUE THREAD

**Issue by RJP43** [RJP43/LiliElbe_EngagedLearners#32]
hey there @joaquindavalosponce, @amandallopez, and @ElleBromer  👋  
  
Your team has been assigned the French-language Hirschfeld chapter.  As a team you will need to associate [the new project schema](https://github.com/RJP43/LiliElbe_EngagedLearners/blob/master/ProjectDocs/LEDA_ODD.rng) to [HirschfeldChapter.xml](https://github.com/RJP43/LiliElbe_EngagedLearners/blob/master/ProjectDocs/archivalMaterials/French_archive/HirschfeldChapter/HirschfeldChapter.xml).
  
Then you will need to correct the XML errors (in the entire file - including errors found in the `<tei:header>` elements). Be sure to reach out via this issue if you hit any snags. Simply tag me - @RJP43 - and add the "help wanted" label to this issue.  
![screenshot highlighting help wanted label](https://user-images.githubusercontent.com/8645521/54369274-a2d2fc00-4643-11e9-8a83-c19196fce6bd.png)  If I do not respond within 48 hours with my 👀 reaction then please shoot me an email at `rparker3 (at) luc (dot) edu` and I will be in contact ASAP. 

After you have uploaded the valid and well-formed XML file in the [Hirschfeld Chapter folder](https://github.com/RJP43/LiliElbe_EngagedLearners/tree/master/ProjectDocs/archivalMaterials/French_archive/HirschfeldChapter), tag me - @RJP43 - and add the "transformation ready" label to this issue.  I will then use our project's XSLT (eXtensible Stylesheet Language Transformation) and will respond to your team here in this issue with an HTML proofing link.   
  
From there your team will work to [HTML proof](https://github.com/RJP43/LiliElbe_EngagedLearners/wiki/Proofing-HTML) the chapter -- reporting errors and design concerns regarding the HTML styling to me via this issue (tagging me and again using the "help wanted" label) and, otherwise, revising the XML where improper encoding has resulted in faulty HTML rendering.  As a reminder, you will use all of the materials in each of the [Hirschfeld Chapter folder](https://github.com/RJP43/LiliElbe_EngagedLearners/tree/master/ProjectDocs/archivalMaterials/French_archive/HirschfeldChapter) (i.e. original transcriptions, facsimile images, etc.) to proof the HTML.
 
Once the chapter has been proofed @ProfPLC and I will inform your project team of next tasks or that your project contribution (well... at least this more "technical" aspect of it) is complete.  Please note the project and the training materials available on this repository are still in flux. This means that there might be moments where I need to inform your team of new updates that will require XML corrections or re-reading of lesson materials; therefore, please check this issue and the general [issues board](https://github.com/RJP43/LiliElbe_EngagedLearners/issues) regularly for alerts from me.   

**_Recent Update_**: We have moved away from using the MIWSchema.rng, and are now using our new schema - [LEDA_ODD.rng](https://github.com/RJP43/LiliElbe_EngagedLearners/blob/master/ProjectDocs/LEDA_ODD.rng).  Please be sure you download this new schema and delete the old one from your computers.  We also now have a much more robust prosopography file listing all of the identified people and places in our project files. Please reference our [LEDA_Prosopography.xml](https://github.com/RJP43/LiliElbe_EngagedLearners/blob/master/ProjectDocs/LEDA_Prosopography.xml) when trying to identify proper key attribute values.  Every xml:id attribute value in the file is a possible key attribute value that can be used in your XML files. We will be outputting an HTML transformation of this prosopography file soon and when we do I will be sure to inform your team via this issue.

**Comment by RJP43** [comment]
:eyes: @joaquindavalosponce, @amandallopez, and @ElleBromer please update your issue with your team's task workflow before the weekend. Also, it looks like @joaquindavalosponce has validated the XML with the new schema and made the corrections. Before transforming, I ask that one of you please go through the file and verify that the structure matches the book chapter template and the translation template from the encoding guidelines as well as the teiHeader template then comment here in this issue when you are ready for the file to be turned into HTML. Thanks and great work so far @joaquindavalosponce! 🎉 👍

**Comment by joaquindavalosponce** [comment]
@RJP43 I have been working so that the structure of the translation matches the french version. Do I have to upload this new corrected word document to the chapter file?

**Comment by RJP43** [comment]
@joaquindavalosponce Yes, please upload any changed documents to your team's folder.   
   
❗️ TEAM, ❗️ it is important for your entire team to be working on these tasks together. @amandallopez and @ElleBromer please update me on this issue with what work you are doing. If it turns out this work is better suited for just one person we can move each of you to another text/group of texts while @joaquindavalosponce works on this chapter.

**Comment by joaquindavalosponce** [comment]
Oh we are working on this together on class right now! Sorry for not being clear @RJP43

**Comment by joaquindavalosponce** [comment]
@RJP43 We just uploaded the new changed document to our team's folder

**Comment by joaquindavalosponce** [comment]
@RJP43 When are you available to meet with us to clarify our doubts?

**Comment by amandallopez** [comment]
Hi, @RJP43! We have been utilizing our class time to map out what needs to be completed and how we want to divvy up the work. However, there is a bit of confusion on proofing the HTML and utilizing Hypothesis to do so. Could you clarify this process for us a bit? Perhaps a meeting won't be necessary, but we are available throughout the week if we still find ourselves amidst confusion. Thank you!

**Comment by RJP43** [comment]
:eyes: @joaquindavalosponce @amandallopez @ElleBromer OKAY! As you continue to work on this text as a team please let me know when you have corrected the schema validation errors and the `<teiHeader>` according to my suggestions laid out in issue #35. Once this is done your file should be ready for HTML transformation so you will assign the issue label (described in the very first comment on this issue) to indicate you are ready. From there I will transform your file and send you a link to the HTML.  

📆 If any of you would like to meet I am available tomorrow in IC study room 218 from noon until 12:30 PM as well as from 1:30 PM to 2:15 PM. I am also available on Thursday at the IC second floor reference desk from 9 AM to 1 PM and from 1 PM to 2 PM in IC study room 219.  Please see [my office hours calendar](http://bit.ly/2MdASUn) for availability updates.

**Comment by RJP43** [comment]
@amandallopez Please alert me with the "transformation ready" label once you have completed updating the chapter's TEI header. Feel free to reach out via this issue and using the "help wanted" label if you run into any snags. Many thanks! 👍 

In response to our meeting in class (this past Tuesday), I have removed @ElleBromer and @joaquindavalosponce from this issue. I will be reassigning the two of you to a new issue as soon as possible. So stay tuned. 😃 Seems there is only enough work left on this text for one person and since @amandallopez is the only one on the team to read French it makes the most sense to keep her on this task. Thank you all for your work so far and see you all in class on Tuesday.

**Comment by amandallopez** [comment]
I went through and assured that the TEI formatting and the person/place encoding was squared away and I believe the document is now transformation ready. However, if something is missing/done incorrectly, let me know and I will fix it. Thank you 👍

**Comment by RJP43** [comment]
:eyes: @amandallopez HUZZAH! 🎉  I will take a look and try running the transformation. I will get back to you, hopefully before class tomorrow, with an HTML proofing link. We'll touch base in class and thanks for add the "transformation ready" label. 👌 Be sure you document all your hard work on the [`Engaged Learning Tracking Hours Spreadsheet`](https://docs.google.com/spreadsheets/d/1VMo0yAKX0qB8dlYiwNMGNdzAQimh-opYzX3XoqJSG2c/edit?usp=sharing).

**Comment by RJP43** [comment]
❗️ 👀 ❗️ @amandallopez I am working on code that will transform the XML for chapters and book sections into our proofing HTML and was relying on your file as well as @Samantha9899's German Foreword to test the transformation. You'll notice I removed the transformation ready label from your issue. This is because there are aspects of your teiHeader element that just will not let it transform. This is because the transformation code is looking for a very particular XML hierarchy based on the encoding guidelines and teiHeader template. I think what is causing the issue is the structure of your title element in your XML; however, looking at your file it seems the entire teiHeader has yet to be corrected. Please see issue #35  for instructions on how to update your teiHeader. I also encourage you to meet with @Samantha9899  or others in the class who have updated their teiHeaders for assistance. The [German Foreword is a good reference file](https://github.com/RJP43/LiliElbe_EngagedLearners/blob/master/ProjectDocs/archivalMaterials/German_archive/LiliForward/LiliForeword.xml).

**Comment by RJP43** [comment]
:eyes: @amandallopez please let me know once you have corrected the teiHeader by remarking your issue with the "transformation ready" and commented below. Thanks! 👍

**Comment by RJP43** [comment]
@amandallopez Since I have not heard from you I have reassigned the HTML proofing of the Hirschfeld chapter to @schola0208 and @bcristiana. Please speak to Pamela regarding your grade in the course. I wish you all the best. 🤓 I am closing this issue since those working on this document will be doing so through HTML. All HTML proofers please reference issue #44 for HTML proofing instructions. 💟
